playbooks/support/scheduling.md

```diff
@@ -107,5 +79,49 @@ To override a shift:
 
    <img width=500 src="images/opsgenie-override-2.png" />
 
-[schedule]: https://artsy.app.opsgenie.com/teams/dashboard/ee381004-a72e-42ef-a733-b350d6693c6c/main
-[artsy-cli]: https://github.com/artsy/artsy-cli
+## **Adding Engineers**
+
+Generally, new engineers should be added to the end of the schedule rotation associated with their time zone. To do this:
```

Comment:
The thinking was that we would not utilize `no one` slots beyond what is laid out in the `Removing Engineers` section. 

>When an engineer leaves:
>- They will be replaced with a no-one placeholder in the schedule as part of the [off-boarding checklist](https://docs.google.com/document/d/10mmqkXnYVp0ZOmF5JwVPyxkzcyIN-rqO8prHgIi11lw/edit).
>- The user’s manager will be asked (in #dev-managers) to override that shift with themselves.
>- Their Opsgenie user account will be deleted.
>
>Responsibility of the departing engineer's manager:
>
>- Once an engineering manager overrides the departing engineer's next shift, they should remove the no-one placeholder from the participants list.

Reply:
> Once an engineering manager overrides the departing engineer's next shift, they should remove the no-one placeholder from the participants list.

This^ must mean once the overridden shift finishes, right? Otherwise, I think removing the `no one` placeholder will shift the schedule, despite the manager's override.

Reply:
Hmm. Somehow missed this. You're correct. It does still shift it down. With that, I agree it does make sense to utilize no-one placeholders, rather than remove them. I'll adjust to include the recommendation made below.